upt_macports/templates/ruby.Portfile

```diff
@@ -0,0 +1,34 @@
+{% extends 'base.Portfile' %}
+
+{% block portgroup %}
+PortGroup           ruby 1.0
+{% endblock %}
+{% block nameversion %}
+ruby.setup          {{ pkg._pkgname() }} {{ pkg.upt_pkg.version }} gem {} rubygems
+ruby.branches       2.5 2.6
+revision            0
+{% endblock %}
+
+{% block versions %}
+{% if pkg.upt_pkg.requirements.run or pkg.upt_pkg.requirements.test %}
+if {${subport} ne ${name}} {
+    {% if pkg.upt_pkg.requirements.run|length > 0 %}
```

Comment:
the "length" part should be removed everywhere in the ruby.Portfile template as well (just as you did for the Perl template)